plugin-script/src/main/java/io/kestra/core/tasks/scripts/AbstractPython.java

```diff
@@ -0,0 +1,28 @@
+package io.kestra.core.tasks.scripts;
+
+import io.kestra.core.exceptions.IllegalVariableEvaluationException;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.runners.RunContext;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Slf4j
+@Deprecated
+public abstract class AbstractPython extends AbstractBash {
+
```

Comment:
It seems odd to me that this task didn't contains the previous properties it has on the core: https://github.com/kestra-io/kestra/blob/develop/core/src/main/java/io/kestra/core/tasks/scripts/AbstractPython.java

Reply:
You are true, the AbstractPython is useless since there props are copied on the Python, I have remove then